scripts/build/Platform/Linux/deploy_cdk_applications.sh

```diff
@@ -61,6 +61,10 @@ then
     exit 1
 fi
 
+echo [cdk_installation] Install nodejs Node.js v17.x
+curl -fsSL https://deb.nodesource.com/setup_17.x | sudo -E bash -
```

Comment:
nit: I wish we could pass in version in setup and not hardcode in .sh and other build scripts.

Reply:
Just updated the script to install the current version instead of hardcoding the version number here.